rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -275,6 +275,19 @@ public boolean checkJobPermissionMethod(String sessionId, String method, String
         }
     }
 
+    @Override
```

Comment:
@fviale Maybe you could simply call the same function as above (jobsInfoListPost(String sessionId, List<String> jobsId))
Because they are exactly the same, no diff at all, only the name changes.
After all, the GET & POST endpoints should behave the same and provide the same outputs for the same input which is a list of job ids. Only the HTTP method changes.

Reply:
you're right, I will update my PR